components/camel-aws/camel-aws2-ddb/pom.xml

```diff
@@ -62,6 +62,12 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-gson</artifactId>
+            <scope>provided</scope>
```

Comment:
I wonder if the other components with this data type transformer has optional for some JARs?
If we dont really do that, then its likely better to just have it normal so everything can work out of the box. And instead of camel-gson, it can dependet on just google-gson

Reply:
I'll fix that. Thanks.